Justfile

```diff
@@ -27,7 +27,9 @@ initramfs $IMAGE: init-work
     #!/usr/bin/env bash
     set -xeuo pipefail
     {{ _ci_grouping }}
-    sudo "${PODMAN}" pull $IMAGE
+
+    # Pull non-local images
+    [[ "$IMAGE" =~ ^localhost/ ]] || sudo "$PODMAN" pull "$IMAGE"
```

Comment:
Can we do a check for "if image not in local storage then pull" instead?

Someone may have ran `sudo podman pull` already, and therefore there's no need to pull again, just because the image doesn't begin with "localhost/"